srcpkgs/ldb/files/disable-compile-error-test.patch

```diff
@@ -0,0 +1,21 @@
+--- tests/ldb_match_test.c
++++ tests/ldb_match_test.c
+@@ -32,6 +32,8 @@
+  * This allows test applications to use custom definitions of C standard
+  * library functions and types.
+  */
++// disabled on musl; fails to compile
++#if 0
```

Comment:
We can change to:
```c
#ifndef __GLIBC__
```
and apply this patch unconditionally.

Reply:
Good point, done.